File: ios/App/App/AppVersion/AppVersionPlugin.m
//
//  AppVersionPlugin.m
//  App
//
//  Created by Eric Horodyski on 11/18/20.
//

#import <Capacitor/Capacitor.h>

CAP_PLUGIN(IONAppVersionPlugin, "AppVersion",
    CAP_PLUGIN_METHOD(getAppVersion, CAPPluginReturnPromise);
)

File: ios/App/App/AppVersion/AppVersionPlugin.swift
//
//  AppVersionPlugin.swift
//  App
//
//  Created by Eric Horodyski on 11/18/20.
//

import Foundation
import Capacitor

@objc(IONAppVersionPlugin)
public class IONAppVersionPlugin: CAPPlugin {
    
    @objc func getAppVersion(_ call: CAPPluginCall) {
        let appVersion = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String ?? ""
        call.success(["value": appVersion])
    }
    
}
